tests/Unit/Framework/MockRuntimeSystem.hpp

```diff
@@ -421,8 +700,18 @@ class MockRuntimeSystem {
   }
 
  private:
-  Parallel::MutableGlobalCache<Metavariables> mutable_cache_{};
-  GlobalCache cache_{};
+  // mock_global_cores[node][local_core] is the global_core.
+  std::vector<std::vector<size_t>> mock_global_cores_{};
```

Comment:
replace mock_global_cores_  and mock_nodes_and_local_cores_ with same types as commented on in MockDistributedObject